future/src/ct/ct_export2pdf.cc

```diff
@@ -49,7 +51,7 @@ void CtExport2Pdf::node_and_subnodes_export_print(const Glib::ustring& pdf_filep
     std::vector<Glib::ustring> tree_pango_slots;
     std::list<CtAnchoredWidget*> tree_widgets;
     Glib::ustring text_font = _pCtMainWin->get_ct_config()->codeFont;
-    _nodes_all_export_print_iter(tree_iter, options, tree_pango_slots, tree_widgets, text_font);
+    _nodes_all_export_print_iter(std::move(tree_iter), options, tree_pango_slots, tree_widgets, text_font);
```

Comment:
why?

Reply:
`tree_iter` is passed by value, without the move it will be copied twice

Reply:
Well, I got it, but it is not optimization and does not improve readability:
- this place is called only once
- it's a single std::move in the whole project
- there are at least two more calls of _nodes_all_export_print_iter where std::move cannot be used

Reply:
And compilers are smart enough to either add std::move themselves, when necessary, or ignore it